### DISCUSSION TITLE
PageContainer组件在支付宝小程序跑不起来

### DISCUSSION THREAD

**Discussion by lanlandemaoya** [NervJS/taro#13806] [Q & A]
export 'PageContainer' (imported as 'PageContainer') was not found in '@tarojs/components'
看官网也是适配的，可是为啥不行呢

**Comment by TheKonka** [comment]
taro 是最新版本吗？

**Comment by lanlandemaoya** [comment]
是3.5.7版本的







引用：




报错：





尝试了各种办法，都不行，依赖删除再安装也不行。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "NervJS/taro"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年5月9日(星期二) 下午5:51
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [NervJS/taro] PageContainer组件在支付宝小程序跑不起来 (Discussion #13806)





 
taro 是最新版本吗？

**Reply by TheKonka** [reply]
升级到最新版本吧，老版本不支持好像

**Comment by lanlandemaoya** [comment]
我升级到最新版本3.6.6了，解决了这个报错问题，可是出现了新的问题，依赖编译错误了

然后我所有页面都打不开了。之前是微信小程序还可以，现在微信小程序跟支付宝小程序都报这个错误了，这个是要怎么解决呢，3.6.6已经是最新版本了吧




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "NervJS/taro"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年5月9日(星期二) 晚上6:54
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [NervJS/taro] PageContainer组件在支付宝小程序跑不起来 (Discussion #13806)





 
升级到最新版本吧，老版本不支持好像

**Reply by TheKonka** [reply]
我这边 3.6.6 是正常的，你报什么错误，能不能贴个报错截图或者复现的仓库呢，或者尝试关闭 prebundle 看下
```jx
	compiler: {
		type: 'webpack5',
		prebundle: {
			enable: false
		}
	},
```

**Comment by lanlandemaoya** [comment]
关闭 prebundle这个之后也是会报错呢
这个是报错信息：






这个是依赖版本：







或者您这边可以远程帮我看看嘛。


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "NervJS/taro"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年5月10日(星期三) 上午9:55
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [NervJS/taro] PageContainer组件在支付宝小程序跑不起来 (Discussion #13806)





 
我这边 3.6.6 是正常的，你报什么错误，能不能贴个报错截图或者复现的仓库呢，或者尝试关闭 prebundle 看下
 	compiler: { 		type: 'webpack5', 		prebundle: { 			enable: false 		} 	},

**Reply by TheKonka** [reply]
![image](https://github.com/NervJS/taro/assets/22173084/9da74a10-24be-417e-bd7a-669cfc0cffa6)
你发的都没发成功吧，都是空白看不到

**Comment by lanlandemaoya** [comment]
啊，我每封邮件都有贴图的，我以为您这边收到了，我这边是看到有的，不然我这边加您QQ？我直接QQ发图片给您。
QQ：1097137116




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "NervJS/taro"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年5月10日(星期三) 上午10:21
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [NervJS/taro] PageContainer组件在支付宝小程序跑不起来 (Discussion #13806)





 

 你发的都没发成功吧，都是空白看不到
